src/engraving/libmscore/score.h

```diff
@@ -1028,8 +1028,9 @@ class Score : public QObject, public ScoreElement
     void setEnableVerticalSpread(bool val);
     qreal maxSystemDistance() const;
 
-    ScoreOrder* scoreOrder() const { return _scoreOrder; }
-    void setScoreOrder(ScoreOrder* order) { _scoreOrder = order; }
+    ScoreOrder scoreOrder() const { return _scoreOrder; }
+    void setScoreOrder(ScoreOrder order) { _scoreOrder = order; }
```

Comment:
Please, move the implementation of those methods to cpp

Reply:
Done